scripts/kube-init.sh

```diff
@@ -39,64 +39,41 @@ else
 fi
 docker --version
 
-# Get the latest stable version of kubernetes
-export K8S_VERSION=$(curl -sS https://storage.googleapis.com/kubernetes-release/release/stable.txt)
-echo "K8S_VERSION : ${K8S_VERSION}"
-
-echo "Starting docker service"
-sudo systemctl enable docker.service
-sudo systemctl start docker.service --ignore-dependencies
-echo "Checking docker service"
-sudo docker ps
-
-echo "Download Kubernetes CLI"
-wget -O kubectl "http://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl"
-sudo chmod +x kubectl
-sudo mv kubectl /usr/local/bin/
-
-echo "Download localkube from minikube project"
-wget -O localkube "https://storage.googleapis.com/minikube/k8sReleases/v1.7.0/localkube-linux-amd64"
-sudo chmod +x localkube
-sudo mv localkube /usr/local/bin/
-
-echo "Starting localkube"
-sudo nohup localkube --logtostderr=true --enable-dns=false > localkube.log 2>&1 &
-
-echo "Waiting for localkube to start..."
-if ! timeout 120 sh -c "while ! curl -ks http://127.0.0.1:8080/ >/dev/null; do sleep 1; done"; then
-    sudo cat localkube.log
-    die $LINENO "localkube did not start"
-fi
+# ref: https://github.com/kubernetes/minikube#linux-ci-installation-which-supports-running-in-a-vm-example-w-kubectl-installation
+
+echo "Home is $HOME"
+export HOME=~
+echo "Home is $HOME"
+
+curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube
+
+export MINIKUBE_WANTUPDATENOTIFICATION=false
+export MINIKUBE_WANTREPORTERRORPROMPT=false
+export MINIKUBE_HOME=$HOME
+export CHANGE_MINIKUBE_NONE_USER=true
+mkdir $HOME/.kube || true
+touch $HOME/.kube/config
```

Comment:
This should not overwrite your existing config, but it is a good suggestion to be on the safe side.